common/utils/works.js

```diff
@@ -87,6 +88,25 @@ export function getDownloadOptionsFromImageUrl(
   ];
 }
 
+export function getCanvases(iiifManifest: IIIFManifest): IIIFCanvas[] {
+  const sequence =
+    iiifManifest.sequences &&
+    iiifManifest.sequences.find(
+      sequence =>
+        sequence['@type'] === 'sc:Sequence' &&
+        sequence.compatibilityHint !== 'displayIfContentUnsupported'
+    );
+  return sequence ? sequence.canvases : [];
+}
+
+export function getManifestViewType(iiifManifest: IIIFManifest) {
+  const canvases = getCanvases(iiifManifest);
+  const downloadOptions = getDownloadOptionsFromManifest(iiifManifest);
+  const pdfRendering =
+    downloadOptions.find(option => option.label === 'Download PDF') || false;
+  return canvases.length > 0 ? 'iiif' : pdfRendering ? 'pdf' : 'none';
```

Comment:
Do we want `none` or can we just make it nullable and thus non-existent and the renderer can deal with it?